src/conversion.rs

```diff
@@ -163,6 +170,7 @@ where
     )
 }
 
+#[cfg_attr(docsrs, doc(cfg(feature = "geo-types")))]
```

Comment:
what does this do?

Reply:
It's a docs.rs directive that calls something out as only being available if a specific feature (in this case `geo-types`) is enabled. I can't find the docs for it anywhere now (of course), but it's used in [quite a few places](https://github.com/search?q=%23%5Bcfg_attr%28docsrs&type=Code).